tests/internal/cluster/cluster.go

```diff
@@ -288,6 +288,22 @@ func SoftRebootSNO(apiClient *clients.Settings, retries uint, interval time.Dura
 	return err
 }
 
+// WaitForClusterUnreachable waits up to timeout for the cluster to become unavailable
+// by attempting to list nodes in the cluster.
+func WaitForClusterUnreachable(client *clients.Settings, timeout time.Duration) error {
```

Comment:
I think this name will look better:
**cluster.WaitForUnreachable**
instead of duplication
**cluster.WaitForClusterUnreachable**

Reply:
Fixed